### ISSUE TITLE
Motorola Moto G 5G plus and GameSir X2 USB C input mapping problems

### ISSUE THREAD

**Issue by JonPita** [hrydgard/ppsspp#16923]
### What happens?
When I try to map inputs for the Gamesir the only inputs that work are ABXY, L1/2 R1/2 and these show up as KBD + codename.
Also when going for automatic control mapping the context menu prompt only shows "virtual" not Gamesir X2 as an option
I tried following the fixes for #15763 but that did not help

### What *should* happen?
Inputs should map with "Gamesir X2 + Dpad" or instead of + Dpad they should be A/B/X/Y, L3, etc.
Automatic mapping should have gamesir X2 as an option, currently virtual is the only option.
Important to note that some other emulators pick the controller up fine like AetherSX2, Duckstation, Drastic, M64 Plus FZ

### What are you using?
PPSSPP Gold latest Version
#### What hardware / device and operating system?
Motorola Moto G 5G Plus Qualcomm® Snapdragon™ 765 5G, Running Android 11
Controller is Gamesir X2 USB C

#### What graphics card (GPU) or mobile phone model?
Qualcomm Adreno 620
#### What PPSSPP version (standalone/official), and did it work before?
It worked fine on a different phone with the same PPSSPP Version
#### Which game or games?

All including main menu

#### Checklist

 - [ ] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed. (dont know how to do this, sorry not adept at github)
 - [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
 - [X] Try resetting settings or older versions and include if the issue is related.
 - [ ] Include logs or screenshots of issue.

**Comment by JonPita** [comment]
Forgot to add my version is 1.14.4 which as far as I know is the latest version

**Comment by Gamemulatorer** [comment]
Duplicate https://github.com/hrydgard/ppsspp/issues/16582?

**Comment by unknownbrackets** [comment]
I wonder if you have some mapping app installed and don't realize it's mapping it?  I think these controllers have one, right?

-[Unknown]

**Comment by JonPita** [comment]
Yeah there is a Gamesir App where you can virtually map, but you have to switch the controllers hardware mode in order for that to happen. 
Also I don't believe it is a dupe of 16582 as the issue is the controllers input not getting recognised, which I believe is also leading to a side effect similar to 16582
